doc/guide/datadir.md

```diff
@@ -0,0 +1,29 @@
+# dataDir
+
+The `dataDir` of these services tend to take *relative* paths, which are usually relative to the project root. As such, when you run these services using `nix run`, their data files are created relative to whichever directory you are in. If you want these data files to always reside relative to the project directory, instead of using `nix run` consider wrapping the process-compose packages in script, via either [mission-control](https://community.flake.parts/mission-control) module or a [justfile](https://just.systems/). `services-flake` uses the latter.
```

Comment:
I'd expect the very first sentence of a guide/*` to explain *what* `dataDir` is.

Reply:
https://github.com/juspay/services-flake/pull/144/commits/37136ca4a0b10050accf0d01eb6d751e20eae146